app/src/main/java/no/usn/mob3000/ui/screens/auth/LoginScreen.kt

```diff
@@ -1,34 +1,116 @@
 package no.usn.mob3000.ui.screens.auth
 
+import androidx.compose.foundation.clickable
 import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
 import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
 import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Button
+import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.Text
+import androidx.compose.material3.TextField
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.unit.dp
 import no.usn.mob3000.R
 import no.usn.mob3000.Viewport
 
+
 /**
- * This is the chessboard page, where users can free-play against an AI or
- * fellow physically near player. (Local multiplayer, in other words).
+ * This is the login screen for authorization to an account
+ * as well as an option for logging in with a guest account.
  *
  * @see ForgotPasswordScreen
  * @see ResetPasswordScreen
- * @author frigvid
- * @created 2024-09-24
+ * @author Anarox
+ * @created 2024-09-30
```

Comment:
The date this function was created was 2024-09-24. Even if you _modify_ it, that doesn't change its creation date, and should be reverted.